tuxemon/event/actions/set_monster_level.py

```diff
@@ -42,7 +52,11 @@ def start(self) -> None:
                 return
 
             monster = self.session.player.monsters[int(monster_slot)]
-            monster.set_level(monster.level + int(monster_level))
+            new_level = monster.level + int(monster_level)
+            monster.set_level(new_level)
+            update_move(monster, new_level)
```

Comment:
Can't we just use the existing monster.update_moves?

Reply:
The existing monster.update_moves? Are you referring to the old piece of code that was inside set_level?
```
            if (
                move.technique not in (m.slug for m in self.moves)
                and move.level_learned <= level
            ):
                self.learn(Technique(move.technique))
```
Do you want that we create a new def update_moves inside monster.py?

Reply:
Can't we use monster.set_moves instead of update_move, which is pretty similar?

Reply:
Unfortunately we can't. 
Let's say someone uses set_monster_level, so it increases the level of rockitten from 10 to 20, 10 levels
Rockitten knows A, B, C techniques
if we use set_moves, then the result would be:
Rockitten knows A, A, B, B, C, C and D, E, F